controllers/nodehealthcheck_controller.go

```diff
@@ -559,6 +559,9 @@ func (r *NodeHealthCheckReconciler) remediate(ctx context.Context, node *v1.Node
 	if err != nil {
 		// An unhealthy node exists, but remediation couldn't be created because lease wasn't obtained
 		if _, isLeaseAlreadyTaken := err.(lease.AlreadyHeldError); isLeaseAlreadyTaken {
+			msg := fmt.Sprintf("Skipped remediation of node: %s,because node lease is already taken", node.GetName())
```

Comment:
NIT: can we add a small space after a comma? IMO it would be easier to read the logs/event this way.


```suggestion
			msg := fmt.Sprintf("Skipped remediation of node: %s, because node lease is already taken", node.GetName())
```